client/my-sites/patterns/components/category-not-found/style.scss

```
@@ -0,0 +1,6 @@
+.is-section-patterns {
+	.empty-content {
+		margin-bottom: 60px;
+		margin-top: 60px;
+	}
```

```suggestion
		min-height: 50vh;
	}
```
It'd be nice to make this slightly taller by default. It looks a bit cramped right now, IMO.

---

That's because the `margin-top` rule is overridden. Anyway, I'll fix that 🙂.